clusterctl/examples/google/machines.yaml.template

```diff
@@ -10,6 +10,8 @@ items:
       value:
         apiVersion: "gceproviderconfig/v1alpha1"
         kind: "GCEMachineProviderConfig"
+        roles:
+        - Master
```

Comment:
It seems like there should be a corresponding deletion on lines 25 and 26

Reply:
some of the common code still uses the roles in the api proper; i'll delete those ones once i fix the common code to no longer depend on roles.